images/mid/Dockerfile

```diff
@@ -124,23 +146,21 @@ COPY jupyter-ompp-proxy/ /opt/jupyter-ompp-proxy/
 # Customize and rebuild omp-ui for jupyter-ompp-proxy install
 # issue with making a relative publicPath https://github.com/quasarframework/quasar/issues/8513
 ARG NODE_OPTIONS=--openssl-legacy-provider
-RUN apt-get update --yes \
-    && apt-get install --yes sqlite3 openmpi-bin libopenmpi-dev\
```

Comment:
im not seeing these packages getting installed in the updated code, did I miss something or were they removed?